src/components/invest/InvestThroughQR.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+export default ({ crowdsaleAddress }) => {
```

Comment:
the component being exported by default should have the same naming as the file that contains it. In this case 'InvestThroughQR', but it may be more appropriately named QRPaymentProcess